run_inrerence.py

```diff
@@ -39,24 +33,41 @@ def parse_args():
         type=float,
         default = 0.2
     )
-
+    parser.add_argument(
+        "--configfile", "-cf",
+        help="Path to json file to load classes and colors",
+        required=True,
```

Comment:
```suggestion
        required=False,
```
With the changes in L54-60